types/testingutils/runner.go

```diff
@@ -184,6 +184,98 @@ var decideRunner = func(consensusInput *types.ConsensusData, height qbft.Height,
 	return v.DutyRunners[types.BNRoleAttester]
 }
 
+// //////////////////////////////// For SSV Tests ////////////////////////////////////////////////////////////////
+var SSVDecidingMsgsForHeight = func(consensusData *types.ConsensusData, msgIdentifier []byte, height qbft.Height, keySet *TestKeySet) []*qbft.SignedMessage {
```

Comment:
what that copied from somewhere else?

Reply:
Before we had the same functionused for both `qbft` and `ssv` tests.
What it did was create set of deciding messages from height 0 to `height`.

The issues for using that in `ssv` are:
1. Now that heights are slots it started to create unintentional amount of messages. Changing the tests.
2. The next deciding message stops the instance of the previous set.